src/index.d.ts

```diff
@@ -1051,31 +1064,128 @@ declare namespace admin.auth {
 
   type TenantType = 'lightweight' | 'full_service';
 
+  /**
+   * Interface representing a tenant configuration.
+   * 
+   * Multi-tenancy support requires Google Cloud's Identity Platform
+   * (GCIP). To learn more about GCIP, including pricing and features,
+   * see the [GCIP documentation](https://cloud.google.com/identity-platform)
+   * 
+   * Before multi-tenancy can be used on a Google Cloud Identity Platform project,
+   * tenants must be allowed on that project via the Cloud Console UI.
+   * 
+   * A tenant configuration provides information such as the type of tenant (lightweight or
+   * full service), display name, tenant identifier and email authentication configuration.
+   * For OIDC/SAML provider configuration management, `TenantAwareAuth` instances should
+   * be used instead. When configuring these providers, note that tenants will inherit
+   * whitelisted domains and authenticated redirect URIs of their parent project.
+   *
+   * All other settings of a tenant will also be inherited. These will need to be managed
+   * from the Cloud Console UI.
+   */
   interface Tenant {
+
+    /**
+     * The current tenant identifier.
```

Comment:
Do you think we should be more specific? like `The current tenant identifier to be passed in the requests.`

Reply:
For the `Tenant` interface, this is only returned in operations like `getTenant`, `createTenant`, `updateTenant` and `listTenants`. It doesn't seem to apply here. I think it applies more to `TenantAwareAuth#tenantId`. Check below to see the changes I added.